core/common/src/RenderSchedule.ts

```diff
@@ -568,6 +568,8 @@ export namespace RenderSchedule {
     public readonly cuttingPlane?: TimelineEntryList<CuttingPlaneEntry, CuttingPlaneEntryProps, CuttingPlane | undefined>;
     /** The total time period represented by this timeline. */
     public readonly duration: Range1d;
+    /** Indicates whether the schedule editing session has been finalized and is no longer active. */
+    public isEditingCommitted: boolean = false;

```

Comment:
The public API shouldn't permit people to modify this property. It's weird anyway that every timeline has a separate `isEditingCommitted` property when really it's a property of the entire script. I suggest making it `@internal`.